### ISSUE TITLE
Change node

### ISSUE THREAD

**Issue by Edgey7** [sologenic/sologenic-expo-wallet#465]
Hi. Every time I log into Sologenic account it is telling me to change node and is not displaying any of my balances. I have tried all the nodes but it stays the same. Please can you advise me on what to do. Thanks

**Comment by SHORTRND** [comment]
Yes, I have the same issue aswell... why isn't any of these issues being brought up not Answered from Solo Support?

**Comment by Lordkoko91** [comment]
Me2. Same Problem.

**Comment by bo10us** [comment]
Same problem with no response from support.  Has anyone worked out the issue? I keep changing nodes but none work.   I have sent xrp to activate it but cannot receive xrp credit without the nodes accepting my deposits????
